src/client/src/orangehrmAdminPlugin/components/LdapSyncConnection.vue

```diff
@@ -62,7 +62,8 @@ export default {
   setup() {
     const http = new APIService(
       window.appGlobal.baseUrl,
-      'api/v2/admin/ldap/sync',
+      // 'api/v2/admin/ldap/sync',
```

Comment:
*

Reply:
I will revert this one since chamara is going to work on this